minpy/nn/solver.py

```diff
@@ -4,10 +4,14 @@
 from __future__ import print_function
 import time
 
+import minpy
+
 from minpy.nn import optim, init
 from minpy import core
 import minpy.numpy as np
 
+from minpy.visualize.writer import SummaryWriter
+import minpy.visualize.summary_ops as summary_ops
```

Comment:
Line 14 will work, line 13 won't. Maybe just leave the remark that uncomment this line when you have installed TensorFlow and want to visualize things?